algolia.gemspec

```
@@ -35,7 +35,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake'
   spec.add_development_dependency 'rubocop', '<= 0.82.0'
 
-  spec.add_dependency 'faraday', ['>= 0.15', '< 2.0']
+  spec.add_dependency 'faraday', ['>= 2', '< 3']
```

We may want to allow older versions here too. If someone has the faraday dependency in their project this range may prevent them from upgrading. WDYT?
```suggestion
  spec.add_dependency 'faraday', ['>= 0.15', '< 3']
```

---

@DevinCodes Faraday 2 is not backward compatible, I cannot use this range (IIRC)